soh/soh/Enhancements/mods.cpp

```diff
@@ -258,6 +260,57 @@ void RegisterRupeeDash() {
     });
 }
 
+struct DayTimeGoldSkulltulas {
+    uint16_t scene;
+    uint16_t room;
+    bool forChild;
+    std::vector<ActorEntry> actorEntries;
+};
+
+using DayTimeGoldSkulltulasList = std::vector<DayTimeGoldSkulltulas>;
+
+void RegisterDaytimeGoldSkultullas() {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnSceneSpawnActors>([]() {
+        if (!CVarGetInteger("gNightGSAlwaysSpawn", 0)) {
+            return;
+        }
+
+        // Gold Skulltulas that are not part of the scene actor list during the day
+        // Actor values copied from the night time scene actor list
+        static const DayTimeGoldSkulltulasList dayTimeGoldSkulltulas = {
+            // Graveyard
+            { SCENE_SPOT02, 1, true,  { {ACTOR_EN_SW, {156, 315, 795}, {16384, -32768, 0}, -20096} } },
+            // ZF
+            { SCENE_SPOT08, 0, true,  { {ACTOR_EN_SW, {-1891, 187, 1911}, {16384, 18022, 0}, -19964} } },
+            // GF
+            { SCENE_SPOT12, 0, false, { {ACTOR_EN_SW, {1598, 999, -2008}, {16384, -16384, 0}, -19198} } },
+            { SCENE_SPOT12, 1, false, { {ACTOR_EN_SW, {3377, 1734, -4935}, {16384, 0, 0}, -19199} } },
+            // Kak
+            { SCENE_SPOT01, 0, false, { {ACTOR_EN_SW, {-18, 540, 1800}, {0, -32768, 0}, -20160} } },
+            { SCENE_SPOT01, 0, true,  { {ACTOR_EN_SW, {-465, 377, -888}, {0, 28217, 0}, -20222},
+                                        {ACTOR_EN_SW, {5, 686, -171}, {0, -32768, 0}, -20220},
+                                        {ACTOR_EN_SW, {324, 270, 905}, {16384, 0, 0}, -20216},
+                                        {ACTOR_EN_SW, {-602, 120, 1120}, {16384, 0, 0}, -20208} } },
+            // LLR
+            { SCENE_SPOT20, 0, true,  { {ACTOR_EN_SW, {-2344, 180, 672}, {16384, 22938, 0}, -29695},
+                                        {ACTOR_EN_SW, {808, 48, 326}, {16384, 0, 0}, -29694},
+                                        {ACTOR_EN_SW, {997, 286, -2698}, {16384, -16384, 0}, -29692} } },
+        };
+
+        for (auto dayTimeGS : dayTimeGoldSkulltulas) {
+            if (IS_DAY && dayTimeGS.forChild == LINK_IS_CHILD &&
+                dayTimeGS.scene == gPlayState->sceneNum &&
+                dayTimeGS.room == gPlayState->roomCtx.curRoom.num) {
+                for (auto actorEntry : dayTimeGS.actorEntries) {
```

Comment:
These for loops could be const refs since you're just reading the data and not modifying them.  That would make this `for (const auto& dayTimeGS` and `for (const auto& actorEntry`.

Reply:
Done.